content/docs/0.14.x/monitoring/prometheus/install/index.md

```diff
@@ -1,38 +1,46 @@
 ---
 title: Install
-description: Setup Prometheus monitoring
+description: Set up Prometheus monitoring
 weight: 1
 icon: setup
 ---
 
-In order to evaluate the quality gates and allow self-healing in production, we have to set up monitoring to get the needed data and fetch the values for the SLIs that are referenced in a SLO configuration.
+In order to evaluate the quality gates and allow self-healing in production, we must set up monitoring to get the needed data and fetch the values for the SLIs that are referenced in an SLO configuration.
 
 
 ## Prerequisites
 
 - Keptn project and at least one onboarded service must be available.
-- Keptn doesn't install or manage Prometheus and its components. Users need to install *Prometheus* and *Prometheus Alert Manager* as a prerequisite.
+- Keptn does not install or manage Prometheus and its components. Users must install *Prometheus* and *Prometheus Alert Manager* as a prerequisite.
 
 ```bash
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 helm install prometheus prometheus-community/prometheus --namespace default
 ```
 
-- The prometheus-service (which will be deployed later) needs access to the Prometheus instance. By default it uses the Prometheus instance running in the cluster. If another Prometheus instance shall be used, create a secret containing the user, password, and url. The secret must have the following format (please note the double-space indentation):
+- The prometheus-service (which will be deployed later) needs access to the Prometheus instance.
+By default it uses the Prometheus instance running in the same name space as Keptn.
```

Comment:
not in the same name space, but on the same Kubernetes cluster in the `monitoring` namespace (fyi, it's "namespace", one word, not "name space")